scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/vqfn.yaml

```diff
@@ -505,3 +505,68 @@ VQFN-32-1EP_5x5mm_P0.5mm_EP3.1x3.1mm:
   #pad_length_addition: 0.5
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
+
+
+VQFN-48-1EP_7x7mm_P0.5mm_EP5.15x5.15mm:
+  device_type: 'VQFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'http://www.ti.com/lit/ds/symlink/cc1312r.pdf'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  # custom_name_format:
+  body_size_x:
+    minimum: 6.9
+    nominal: 7
+    maximum: 7.1
+  body_size_y:
+    minimum: 6.9
+    nominal: 7
+    maximum: 7.1
+  overall_height:
+    #minimum: 0.8
+    #nominal: 0.85
+    maximum: 1
+  lead_width:
+    #minimum: 0.18
+    nominal: 0.24
+    #maximum: 0.3
+  lead_len:
+    #minimum: 0.25
+    nominal: 0.3
+    #maximum: 0.35
+
+  EP_size_x:
+    minimum: 5.05
+    nominal: 5.15
+    maximum: 5.25
+  EP_size_y:
+    minimum: 5.05
+    nominal: 5.15
+    maximum: 5.25
+  # EP_paste_coverage: 0.65
+  EP_num_paste_pads: [2, 2]
+  # heel_reduction: 0.05 #for relatively large EP pads (increase clearance)
+
+  thermal_vias:
+    count: [3, 3]
+    drill: 0.33
+    # min_annular_ring: 0.15
+    paste_via_clearance: 0.1
+    # EP_num_paste_pads: [2, 2]
+    # paste_between_vias: 1
+    # paste_rings_outside: 1
+    EP_paste_coverage: 0.7
+    grid: [1.2, 1.2]
```

Comment:
datasheet suggests a very strange layout not supported by the script so i would suggest to drop this and let the script decide automatically how to place vias.